libs/coin-modules/coin-canton/src/bridge/sync.integ.test.ts

```diff
@@ -129,16 +129,21 @@ describe("sync (devnet)", () => {
       mockGetBalance.mockResolvedValue([
         {
           instrument_id: "Amulet",
-          amount: 1000000,
+          amount: "500",
+          locked: true,
```

Comment:
this should be locked: false according to expectations, no ?

Reply:
yes correct, fixed